.betterer.results

```diff
@@ -6659,16 +6659,21 @@ exports[`better eslint`] = {
       [0, 0, 0, "Do not use any type assertions.", "1"],
       [0, 0, 0, "Unexpected any. Specify a different type.", "2"]
     ],
+    "public/app/plugins/datasource/cloudwatch/components/ConfigEditor.test.tsx:5381": [
+      [0, 0, 0, "Unexpected any. Specify a different type.", "0"]
```

Comment:
I'm not terribly familiar with betterer as a tool, but I'm guessing this means that we're adding an "any" when we probably should not right?

Reply:
probably, but it's in a test and I'm not sure what alternate thing I would do, so I left it.